src/main/java/seedu/address/model/person/NameContainsKeywordsPredicate.java

```diff
@@ -18,7 +16,7 @@ public NameContainsKeywordsPredicate(List<String> keywords) {
     @Override
     public boolean test(Person person) {
         return keywords.stream()
-                .anyMatch(keyword -> StringUtil.containsWordIgnoreCase(person.getName().getFullName(), keyword));
+                .anyMatch(keyword -> person.getName().getFullName().toLowerCase().contains(keyword.toLowerCase()));
```

Comment:
Not too sure why the previous version does not work but trust that this works as intended